### DISCUSSION TITLE
Question: Revision/Versions are being saved for every character typed on Etherpad

### DISCUSSION THREAD

**Discussion by ytiwari** [ether/etherpad-lite#5230] [Q&A] [answered]
Hi

Etherpad saves a version for each character typed. Can we restrict or set a time interval for capturing the version, rather than making it on every character. Please find attached the recorded screen referring to the same issue.

https://user-images.githubusercontent.com/1473339/137266606-ec687206-ec76-48be-992a-bf6f1838992f.mp4

Regards,

**Comment by webzwo0i** [comment, marked as answer]
A short interval is necessary for keeping all connected clients in sync. The changeset library won't complain if larger changesets (e.g. after 5 or 15 seconds) are transmitted, but usability will suffer because other clients won't immediately get the changes.
The default is one revision every 500ms or something like that, so it's not really "every character" unless typing slowly.

**Comment by ytiwari** [comment]
Thank you so much for your quick and detailed response.
